src/mixins/getParticipants.js

```diff
@@ -79,7 +79,7 @@ const getParticipants = {
 		},
 
 		debounceUpdateParticipants() {
-			if (!this.isActive) {
+			if (!this.isActive && !this.isInCall) {
```

Comment:
but also the `!this.isActive` is problematic.
Should we store the timestamp of the last refresh and in a watcher on `isActive` compare and in case it's "30+ seconds" trigger an update?

Reply:
Changes can happen within 30s and thus they won't be reflected. 
Always triggering an update when isActive becomes true would be better as no actual update happened when the tab is inactive ?

Reply:
Looks perfect now!